### DISCUSSION TITLE
Dawn Breaker Prizes

### DISCUSSION THREAD

**Discussion by Manu-M123** [sunflower-land/sunflower-land#2530] [Ideas]
**lack of incentive to burn all resources on lanterns.**

I've noticed that big crop/resource holders don't feel incentivized to burn all their resources for a very low percentage of gaining a hoot.

For example, if I have 1 million sunflowers, I can make 142 lanterns. 
And suppose that the weekly goal of 25,000 lanterns is reached.
Well, if I burned 1 million sunflowers, lots of sfl and wood, I would only have a 0.568% chance of earning a hoot, if there are 10 hoots, then I have a 5.68% chance of winning a hoot.
And no one would burn so many resources knowing that they have a 95% chance of losing :/

That is why it is necessary to encourage more, as was done in the last war, that the top 10 of each week received prizes and there was also a lottery.

**My idea is that:**

--> The top 3 of each week will win a hoot as a prize, and that the other 7 remaining will be raffled.
--> Or that the top 10 of the 8 weeks together get a hoot.

This would generate a healthy mini battle to see who burns more resources and who stays with the top.
So, me and many more people, if they would find sense to burn lots of resources.

If your goal is to burn resources you should consider this mechanic.

What do you think?

**Comment by TheDcol** [comment]
The war event was so excited due to the top 10 rewards.

I think it's not a bad idea :)

Would definitely tempt me in burning..

**Reply by albi025b** [reply]
Yee me too.
He covered all points! 
Hope they look into it.

**Reply by Manu-M123** [reply]
> Si yo también. ¡Cubrió todos los puntos! Espero que lo investiguen.

yep. I think I should discuss it quickly to notify the community if this is implemented, knowing that this week has less than 3 days left

**Comment by poteitouweb3** [comment]
I think it's not good.

If you burn the max resources you can to get top 10 and stay in until the end of the week so well done, you reached it, congrats. But if you don't?

With raffle you'll have at least the probability to won as everyone have but if you lost your top 10 after burn all you can it would be very frustrating so i prefer to stay with raffle even knowing that i crafting 23 lanterns and a guy that crafted only 5 have almost the same chance to get a Hoot.

**Reply by Manu-M123** [reply]
If you want to fight to be in the top 10 you have to do the calculations and see if you have the resources to compete against the "whales", otherwise you wouldn't risk it. 

The same thing happened in the last war, there will always be a top 11. As far as I can remember, no one complained about being left out of the top 10.

**Reply by poteitouweb3** [reply]
Well.. I don't have the resources but i can make a 'cost benefit' and see if worth to spend real money for that. Just trust , FUD is around

**Comment by adamhannigan** [comment]
That is fair feedback. We will continue this first week as promised, but going forward we will see how we can incorporate this idea. 

If you are grinding heavily, it does make sense to be rewarded additionally ❤️

**Reply by poteitouweb3** [reply]
Adam, there is only one thing to consider and that is the dissatisfaction of those who will be absolutely certain that they won't have the chance to get a Hoot. The game has come out of a culture of only getting the best NFTs who buy the most resource, and by raffling off the 100 Hoots to those who spend the most, we are returning to exactly that point once again.

"Want a Hoot? Well, enter the battle and spend ~300 dollars in resources for a chance." It doesn't seem very in line with the game's purpose, and we genuinely know that no one will get top 10 weekly without investing money in it.

**Reply by albi025b** [reply]
But it doesn’t have to be an either or. You could give the top 3-5 players a hoot, the top 10-15 players some kinda rare decoration (contribution banner, bear maybe some fix sfl)

and the other 5-7 hoots goes randomly to all other players who crafted lanterns. 

If you do it like that you make both happy. 
And the whales are happy to burn a lot more ressources. I think a small competition between the whales is important. Because fact is whales have the most ressources. But I’m only to willing burn like 400$ worth of ressources if I know I’m getting a hoot or other rare nft when I get into the top “3-10” players. 
And I’m not willing to burn a lot more ressources for only a small increased percentage of winning one if I’m lucky . 

I think this feature is there to burn ressources. And if u want the whales to burn resources or even buy more on market you have to give the something (not based on luck). Look at the war. It worked great there with rewarding the top 10 players! Stick to something that allready worked really good in the past .

**Reply by VZozul** [reply]
Why is everything so unfair in this game? Why can't everything be announced at once? Why did I burn more than $200 and get nothing in return? If I had known, I would have left this money for the second week, but no, no one told us anything. So even on the second week, you don't need SFL. Why did I burn so much, but the one who bought 1 ticket won? Why couldn't the chance be made proportionally, because every subsequent Lantern cost more, so why wasn't the weight of this ticket bigger? Do we support multi-accounts again? Then it was necessary to make the price the same, not increasing. And so it turns out that you simply threw those who burned a lot. I bought 50 Lanterns, do you think I want to play further after that? You talk about balance, you do balance, but at the same time you make the quest random, and for example, I got 5 gold in a quest, and after 2 short quests, I got 5 gold again. You have to wait 24 hours, and during this time, someone was lucky and completed 10 quests during this time. What was the point of removing RNG in the game and adding it again? You want to burn resources now, so make it fair, because those who burned the most, let's say the top 10, are the people who feed you, and you make them think about selling everything and leaving this game.

**Reply by VZozul** [reply]
Honestly, after such a throw, I'll wait for Bean, then the prices will rise, I'll sell everything and that's it. You should think about what you are doing. And before doing something, think everything through, and not change the rules of the game all the time. The truth is, was it impossible to make the weight of the ticket proportional to the difficulty, so that when you craft the 50th Lantern, it would not be the same weight as the 1st? Or at least transfer the purchased lanterns to the next week? So that those who burned a lot and did not win, could get more chances next week.

**Comment by rasheed19871** [comment]
I don't think so the top player should recieve directly the first price because i think that sunflower may have a few whale and they will burn happily what about the rest of the player which may be 90 percent are not whale, 
i think the whale should be rewarded but not giving the main price which is the nft, i have seven sunflower friend i invited and they are not whale even then because of the nft we aim to get atleast 25 lantern so we have a chance plz don't give directly the nft give them some more price

**Reply by albi025b** [reply]
This is why you could give the top 3-5 players a hoot and some sfl. The rest 5-7 hoots go randomly to all other players. In the war it worked really well to give the top 10 players some rewards.->  “Go with what worked in the past.”

I think the intention behind this lantern feature is to burn a lot of ressources and SFL. And I think the most ressources a currently owned by whales. 
So if you want the whales to burn a lot of ressources you have to give them higher chance to get something in return ( and not just a really small percentage for owning 20 more lanterns which is like 0,001%) … otherwise I’m not willing to spend much more ressources then 15 lanterns. 

But if I  know I’m  getting the hoot if I’m hitting the top 3 I’m willing to burn a loooot more. 
So if you implement some competition for the whales they will burn a loooooot more ressources which is the intention of this lantern feature I suppose.

**Comment by rasheed19871** [comment]
You got good point,I think people that burn a lot resources and get nothing should be given some incentive

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
